cmd/kpromo/cmd/pr/pr.go

```diff
@@ -189,40 +189,75 @@ func runPromote(opts *promoteOptions) error {
 	branchname := opts.project + "-" + opts.tags[0] + promotionBranchSuffix
 
 	// Get the github org and repo from the fork slug
+	repo, err := prepareRepository(branchname, opts)
+	if err != nil {
+		return err
+	}
+
+	defer func() {
+		if mustRun(opts, "Clean fork directory?") {
+			err = repo.Cleanup()
+		} else {
+			logrus.Infof("All modified files will be left untouched in %s", repo.Dir())
+		}
+	}()
+
+	// Handle manifests
+	modified, err := handleManifests(ctx, repo, opts)
```

Comment:
`handleManifests` is a bit broad name, what do you think about calling it `growManifest` instead?